### PR TITLE
feat(runtests, tests/, verification/): create view for sending verifi…

### PR DESCRIPTION
**Pull request by drshrey**
…cation and tests

### CODE REVIEW

File: tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+"""
+import random
+
+# Third Party Stuff
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from rest_framework.test import APITestCase, APIClient
+# from rest_framework import serializers
+from nose.tools import eq_, raises
+# from rest_framework.serializers import ValidationError
```
Comment (glemmaPaul) [inline tests/test_views.py]: Maybe remove these commented import

File: verification/views.py [hunk 2]
```diff
@@ -14,4 +20,21 @@ def get_context_data(self, code=None, *args, **kwargs):
...
+        return context
+
+
+class SendVerificationAPIView(generics.CreateAPIView):
+    serializer_class = VerificationSerializer
+
+    def post(self, request, verification_type=None):
+        try:
+            service = get_service(verification_type)
+        except Exception as e:
```
Comment (glemmaPaul) [inline verification/views.py]: Maybe only except on `ValueError`

File: tests/test_views.py (hunk 1 above)
Comment (drshrey) [inline tests/test_views.py]: sounds good

File: verification/views.py (hunk 2 above)
Comment (drshrey) [inline verification/views.py]: true, good point

**Comment by glemmaPaul** [comment]
LGTM ⚓

**Pull request merged**